src/gbm/gblinear.cc

```diff
@@ -219,6 +221,24 @@ class GBLinear : public GradientBooster {
     return model_.DumpModel(fmap, with_stats, format);
   }
 
+  void FeatureScore(std::string const &importance_type,
```

Comment:
Was the bias term ignored before? I don't want to break existing behaviour, but this term is informative.

Reply:
It wasn't included before in R.  This is new for Python as it wasn't supported before.